bigtable/snippets/writes/writes_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+from google.cloud import bigtable
+from write_batch import write_batch
+from write_conditionally import write_conditional
+from write_increment import write_increment
+from write_simple import write_simple
+
+PROJECT = os.environ['GCLOUD_PROJECT']
+BIGTABLE_INSTANCE = os.environ['BIGTABLE_CLUSTER']
+TABLE_ID_PREFIX = 'mobile-time-series-{}'
+
+
+def test_writes(capsys):
+    client = bigtable.Client(project=PROJECT, admin=True)
+    instance = client.instance(BIGTABLE_INSTANCE)
+
+    table_id = TABLE_ID_PREFIX.format(str(uuid.uuid4())[:16])
+    table = instance.table(table_id)
+    if table.exists():
+        table.delete()
+
+    column_family_id = 'stats_summary'
+    column_families = {column_family_id: None}
+    table.create(column_families=column_families)
```

Comment:
Use a pytest fixture with `yield` so that the table is cleaned up even if the tests fail.

https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code

Reply:
Not sure how to do this with what I have. I put `@pytest.fixture(scope="module")` before def tests_writes, but that seems to break things and then I'm not sure what else to do.

Reply:
That's not how you use fixtures...

An example fixture:

https://github.com/googleapis/google-cloud-python/blob/0e231cab5f066ae5b3e254d0cb7d204e3dac7ebc/bigquery/samples/tests/conftest.py#L58-L68

Note: it's in conftest.py, but could just as easily be in the same file as the tests.

To use, simply add an argument to your test function with the same name as the fixture function.

https://github.com/googleapis/google-cloud-python/blob/0e231cab5f066ae5b3e254d0cb7d204e3dac7ebc/bigquery/samples/tests/test_list_tables.py#L18

Reply:
I'm don't really understand this still. These examples all look different. Just committed what I understood

Reply:
OK. I just pushed 3233a426a27180d88621b0750914c41c47ba8208 with my requested changes.

The gist of "fixtures" are that they are the equivalent of a `setUp` function in JUnit. If you `yield` from a fixture, the remaining piece of the function is equivalent to `tearDown` in JUnit.